dags/merino_jobs.py

```diff
@@ -0,0 +1,76 @@
+import datetime
+from typing import Any, Dict, List
+
+from airflow import DAG
+from airflow.hooks.base import BaseHook
+
+from dags.operators.gcp_container_operator import GKEPodOperator
+from dags.utils.tags import Tag
+
+DOCS = """\
+    Merino Jobs
+
+    Dag for orchestrating jobs that build datasets that are used in merino.
+    The jobs are run via the GKEPodOperator 
+"""
+
+def merino_job(name: str, arguments: List[str], env_vars: Dict[str, Any]):
+    return GKEPodOperator(
+        task_id=name,
+        name=name,
+        image="mozilla/merino-py:latest",
+        project_id="moz-fx-data-airflow-gke-prod",
+        gcp_conn_id="google_cloud_airflow_gke",
+        cluster_name="workloads-prod-v1",
+        location="us-west1",
+        cmds=["python", "-m", "merino.jobs.cli"],
+        arguments=arguments,
+        env_vars=env_vars,
+        email=[
+            "wstuckey@mozilla.com",
+        ],
+
+    )
+
+default_args = {
+    "owner": "wstuckey@mozilla.com",
+    "start_date": datetime.datetime(2019, 7, 25),
+    "email": ["wstuckey@mozilla.com"],
+    "email_on_failure": True,
+    "email_on_retry": True,
```

Comment:
I just want to confirm that we're OK with having all failure alerts go to `wstuckey@mozilla.com`? If DISCO team will be owning this, then maybe it would make more sense to have these sent to a mailing list with team members (including Wil). Tagging @ncloudioj for his thoughts on this.

Reply:
i only have myself here in case there's permission errors as we stand this up. in that case i didn't want to spam a big list. we can have a follow up to add a broader team mailing list once it's running.

Reply:
> we can have a follow up to add a broader team mailing list once it's running.

OK! That sounds good too!

Reply:
We can use `conserv-alert at mozilla.com` for this one.